src/bentoml/_internal/frameworks/diffusers_runners/utils.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import annotations
+
+import logging
+
+import diffusers
+import inflection
```

Comment:
We probably need to add inflection to pyproject.toml dependencies here

Reply:
Actually if this is light enough should just vendor the whole file into utils.py?